students/kimjy/westagram/users/check_items.py

```diff
@@ -0,0 +1,43 @@
+import re 
+from varname import nameof
+
+email_policy = [
+        '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$',
+        'email_policy'
+]
+password_policy = [
+        "((?=.*[0-9])(?=.*[a-z|A-Z])(?=.*[\`\~\!\@\#\$\%\^\&\*\(\)\-\_\=\+\\\|\;\:\'\"\,\<\.\>\/\?]).{8,})",
+        'password_policy'
+]
+login_key_list = [
+        'username', 
+        'email', 
+        'password', 
+]
+
```

Comment:
```suggestion
import re

from django.core.exceptions import ValidationError

def validate_email(value) :
    email_regex = re.compile('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$')
    
    if not email_regex.match(value):
        raise ValidationError('INVALID_EMAIL_ADDRESS')

def validate_password(value) :
    password_regex = re.compile("^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$")
    
    if not password_regex.match(value):
        raise ValidationError('INVALID_PASSWORD')
```
validator.py의 가장 기본 구조 입니다.
비교해 보시고 python 다운 코드가 무엇일지 생각해보면 좋을 것 같아요.

구두로 말씀 드렸듯이 flag을 만들어 조건에 따라 flag을 변화 시키는 것은 C언어 같은 스타일입니다.
flag 없이도 충분히 구현 할 수 있도록 짜는 것이 좀 더 python 다운 코드라고 할 수 있을 것 같습니다.

Reply:
코멘트 감사합니다.
flag의 경우에는 제가 익숙했던 방법이어서 사용했던 것 같습니다. 파이썬에서는 사용하지 않도록 주의하겠습니다.
말씀해주신 의견 반영하여 수정하였습니다.